scss/atoms/typography/_typography.scss

```diff
@@ -21,40 +21,45 @@ body {
 }
 
 h1 {
-	font-size: em( 67px ); // 54px works nicely, as well
-	font-weight: 700;
-	line-height: 1;
+	@include heading($h1);
 }
 
 h2 {
-	font-size: em( 36px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h2);
 }
 
 h3 {
-	font-size: em( 30px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h3);
 }
 
 h4 {
-	font-size: em( 24px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h4);
 }
 
 h5 {
-	font-size: em( 20px );
-	font-weight: 600;
-	line-height: 1;
+	@include heading($h5);
 }
 
 h6 {
-	font-size: em( 16px );
-	font-weight: 600;
-	line-height: 1;
-	text-transform: uppercase;
+	@include heading($h6);
+}
+
+p {
+	text-align: map-get($p,textAlignment);
+	margin: em(map-get($p,marginTopBottom)) em(map-get($p,marginLeftRight));
+	line-height: 1.5;
+	font-size: em(map-get($p,fontSize));
+}
+
+ul, ol {
+	margin: 0 0 10px;
+}
+
+ul ul,
+ol ul,
+ul ol,
+ol ol {
+  	margin: 0;
```

Comment:
still something wrong with the indentation. No spaces. Single tab. Turn off soft tabs in your text editor.